src/main/java/view/SubView.java

```diff
@@ -67,7 +69,7 @@ public void actionPerformed(ActionEvent evt) {
                         if (evt.getSource().equals(search)) {
                             // System.out.println("Search Button pressed");
                             // subViewPresenter.prepareSearchView();
-                            subViewController.execute("recommendation");
+                            subViewController.execute("search");
```

Comment:
When search is pressed, the SearchView has issues in it. Will take a look and try to fix this bug soon.